draft-ietf-wimse-s2s-protocol.md

```diff
@@ -313,6 +313,8 @@ A WPT contains the following:
     * `exp`: The expiration time of the WIT (as defined in {{Section 4.1.4 of RFC7519}}). WPT lifetimes MUST be short,
      e.g., on the order of minutes or seconds.
     * `jti`: A unique identifier for the token.
+    * `wth`: Hash of the Workload Identity Token. The value, as defined in {{to-wit}}, is the base64url encoding of the SHA-256
+     hash of the ASCII encoding of the token's value.
```

Comment:
```suggestion
    * `wth`: Hash of the Workload Identity Token, defined in {{to-wit}}. The value is the base64url encoding of the SHA-256
     hash of the ASCII encoding of the token's value.
```

The token is defined in {{to-wit}}, not the hash.

Reply:
Yaron's suggestion looks good to me.